app/actors/file_set_attach_files_actor.rb

```diff
@@ -0,0 +1,33 @@
+# Actions are decoupled from controller logic so that they may be called from a controller or a background job.
+class FileSetAttachFilesActor < Hyrax::Actors::FileSetActor
```

Comment:
I think we just extend/override the Hyrax::Actors::FileSetActor to accept a relation/file use parameter.